src/Pieces/Knight.cpp

```diff
@@ -0,0 +1,21 @@
+#include "../include/Pieces/Knight.h"
```

Comment:
Don't use relative includes. It makes the project hard to maintain, for example when you want to move headers between places. You should configure your CMake so that headers can be universal.
In this case - #include "../include/Pieces/..." will become the more elegant #include "Pieces/Knight.h
Check out this explanation: https://discourse.cmake.org/t/target-include-directories-vs-headers-in-target-sources/2625/2